examples/xkcd-bot/main.rs

```
@@ -91,12 +90,6 @@ async fn runner(mut shard: Shard, client: Arc<Client>) {
         let event = match item {
             Ok(Event::GatewayClose(_)) if SHUTDOWN.load(Ordering::Relaxed) => break,
             Ok(event) => event,
-            Err(error)
```

Ditto this- why is it removed? doesn't seem to have things to do with this, unless it's fixing a lint I'd open this PR seperately

---

The code won't compile otherwise as Twilight 0.16 swallows websocket errors now, and #36 didn't address this I assumed was an oversight. But yeah, it might be better suited for a separate PR to fix the example bot code.

---

Interesting. I wonder how that passed CI? It does check examples.